main.nf

```diff
@@ -71,6 +71,9 @@ def helpMessage() {
       --pon_index              [file] Index of pon panel-of-normals VCF
       --ascat_ploidy            [int] Use this parameter together with to overwrite default behavior from ASCAT regarding ploidy. Note: Also requires that --ascat_purity is set.
       --ascat_purity            [int] Use this parameter to overwrite default behavior from ASCAT regarding purity. Note: Also requires that --ascat_ploidy is set.
+      --umi                       If provided, UMIs steps will be run to extract and annotate the reads with UMI and create consensus reads
+      --read_structure1           When reads contain UMIs a structure for read 1 should be provided, to allow removal of UMI sequence from the read. See: https://github.com/fulcrumgenomics/fgbio/wiki/Read-Structures
```

Comment:
```suggestion
      --read_structure1      [string] When reads contain UMIs a structure for read 1 should be provided, to allow removal of UMI sequence from the read.
                                                    See: https://github.com/fulcrumgenomics/fgbio/wiki/Read-Structures
```

Reply:
when processing UMIs a read structure always needs to be provided, because the software requires a structure for each of the fastq. should the read not contain any UMI, the format would be just +T (i.e. all template, any length)
it's good to make this explicit and can be added to documentation as well